### DISCUSSION TITLE
allow addresses to be put on entrances and barrier=gate/entrance

### DISCUSSION THREAD

**Discussion by dieterdreist** [streetcomplete/StreetComplete#4566] [Feedback]
It would be marvelous if one could add addresses on entrances and barrier=gate/entrance. This is the generally used method here, but is at least occasionally also used in many other places

**Comment by valerio-bozzolan** [comment]
+1 (↑ Here: Italy)

**Comment by westnordost** [comment]
I'm working on it.  I plan it so that it is also possible to add new entrances to buildings directly with addresses on it instead of only free-floating addresses. (EveryDoor already has this capability)
